tools/topology/m4/buffer.m4

```diff
@@ -29,4 +29,6 @@ define(`W_BUFFER',
 dnl COMP_BUFFER_SIZE( num_periods, sample_size, channels, fmames)
 define(`COMP_BUFFER_SIZE', `eval(`$1 * $2 * $3 * $4')')
 
+define(`COMP_BUFFER_FRAMES', `eval(`($1 / 1000) * ($2 / 1000)')')
```

Comment:
the naming 'COMP_BUFFER_FRAMES' is somewhat hard for understanding, how about 'COMP_PERIOD_FRAMES'?

Reply:
@keyonjie sure I can do it if it is more clear... just a little bit error prone search-replace-regexp for me... if not absolutely needed, I wouldn't do it :)

Reply:
BTW, how about the case scheduling times smaller than 1000us? And add some comment about the macro also.
Can we change it to:
```
dnl COMP_PERIOD_SIZE( sample_rate, period_us)
define(`COMP_PERIOD_FRAMES', `eval(`($1 * $2) / 1000000')')
```

Reply:
@keyonjie yes that looks better, I can change for sure.